src/useSize.tsx

```diff
@@ -67,7 +67,7 @@ const useSize = (
     }
 
     return () => {
-      if (window) {
```

Comment:
I'm not sure I understand how this bug happens. Is `<iframe>` removed from DOM before this line executes?